### ISSUE TITLE
[FEATURE] Add `class-spacing` rule

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#464]
**Is your feature request related to a problem? Please describe.**
It would be helpful to add a `no-extra-class-space` rule that disallows unnecessary extra spaces in the class attribute.

This rule would report cases where class names are separated by multiple spaces or where leading/trailing spaces exist, as they provide no semantic value and may cause inconsistency or confusion.

❌ Incorrect
```html
<div class="foo  bar"></div>
<div class=" foo bar"></div>
<div class="foo bar "></div>
<div class="foo   bar   baz"></div>
```
✅ Correct

```html
<div class="foo bar"></div>
```
**Additional context**
